kuiper-game/Services/SolarSystemService.cs

```diff
@@ -1,21 +1,76 @@
+using System;
+using System.Linq;
+using Kuiper.Domain;
+using Kuiper.Systems;
 using System.Collections.Generic;
 using Kuiper.Domain.CelestialBodies;
 using Kuiper.Repositories;
-using System.Linq;
 using System.Numerics;
-using System;
 
 namespace Kuiper.Services
 {
     public class SolarSystemService : ISolarSystemService
     {
+        private SolarSystem _currentSystem;
+        private List<CelestialBody> SolarSystem;
+        public SolarSystemService(ISolarSystemRepository repository)
+        {
+            SolarSystem = new List<CelestialBody>();
+            SolarSystem = CreateSolarSystem(repository).ToList();
+        }
+
+        private IEnumerable<CelestialBody> CreateSolarSystem(ISolarSystemRepository repository) {
+            return repository.GetSolarSystem();
+        }
+        public SolarSystem SetupGame()
```

Comment:
I really don't like this.
I don't think it's the job of the SolarSystemService to setup the game.
Perhaps there's another service for that, which uses SolarSystemService.

Reply:
I agree, but this is the hammer right now

Reply:
I suggest #18 to find the right solution